.evergreen/config.yml

```diff
@@ -386,6 +386,20 @@ tasks:
   - func: "fetch source"
   - func: "run clang-tidy"
 
+- name: calc-release-version-selftest
```

Comment:
Similar comment as https://github.com/mongodb/mongo-cxx-driver/pull/979#discussion_r1232516780. Suggest removing from CI and release process.